test/param.c

```diff
@@ -1528,7 +1531,12 @@ ODBC_TEST(odbc45)
   for (i= 0; i<sizeof(XpctdValue); ++i)
   {
     CHECK_STMT_RC(Stmt, SQLFetch(Stmt));
+  #if defined(__s390x__)
+    SQLGetData(Stmt, 1, SQL_C_BIT, &value, sizeof(value), 0);
+    is_num(value, XpctdValue[i]);
+  #else  
```

Comment:
I also suspect, that explicit casting of XpctdValue[i] to int could fix the test for s390. Not sure, though